app/src/components/misc/SupportPanel/index.tsx

```diff
@@ -0,0 +1,196 @@
+  const isSupportChatOpened = useSelector(getIsSupportChatOpened);
+  const supportItems = useMemo(
+    () =>
+      [
+        slackConnect &&
+          slackInviteVisibilityStatus && {
+            key: "1",
+            label: (
+              <div
+                className="support-panel-item-style"
+                onClick={() => {
+                  setVisible(false);
+                  sendSlackInvite();
+                  trackSupportOptionClicked("slack");
+                  trackEvent("join_slack_connect_clicked", {
+                    source: "support_option",
+                  });
+                }}
+              >
+                <span className="support-panel-label-style">Ask us on Slack</span>
+              </div>
+            ),
+            icon: <SlackOutlined className="support-panel-icon-style" />,
+          },
+        {
+          key: "2",
+          label: (
+            <a
+              href="https://www.reddit.com/r/requestly/"
+              target="_blank"
+              rel="noreferrer"
+              className="support-panel-item-style"
+              onClick={() => {
+                setVisible(false);
+                trackSupportOptionClicked("reddit");
+              }}
+            >
+              <span className="support-panel-label-style">Ask us on Reddit</span>
+            </a>
+          ),
+          icon: <PiRedditLogo className="support-panel-icon-style" />,
+        },
+        {
+          key: "3",
+          label: (
+            <div
+              className="support-panel-item-style"
+              onClick={() => {
+                setVisible(false);
+                setIsRequestBotVisible(true);
+                trackSupportOptionClicked("ask_ai");
+              }}
+            >
+              <span className="support-panel-label-style">
+                Ask AI<i style={{ color: "#bbbbbb" }}> 24x7 Support</i>
+              </span>
+            </div>
+          ),
+          icon: <BotIcon className="support-panel-icon-style" />,
+        },
+        {
+          key: "4",
+          label: (
+            <div
+              className={`support-panel-item-style ${!paidUser && "support-panel-item-disabled"}`}
+              onClick={() => {
+                if (paidUser) {
+                  window.$crisp.push(["do", "chat:open"]);
+                  trackSupportOptionClicked("chat_with_us");
+                } else {
+                  dispatch(
+                    // @ts-ignore
+                    actions.toggleActiveModal({
+                      modalName: "pricingModal",
+                      newValue: true,
+                      newProps: { selectedPlan: null, source: "support_option_clicked" },
+                    })
+                  );
+                  trackSupportOptionClicked("upgrade_to_chat_with_us");
+                }
+                paidUser
+                  ? window.$crisp.push(["do", "chat:open"])
```

Comment:
https://github.com/requestly/requestly/pull/1968#discussion_r1684086883
```suggestion
                  ? window?.$crisp?.push(["do", "chat:open"])
```